lib/index.ts

```diff
@@ -4,6 +4,9 @@ export {
   isValidationError,
   isValidationErrorLike,
   fromZodError,
+  fromZodIssue,
+  type ZodError,
+  type ZodIssue,
+  type FromZodErrorOptions,
+  type FromZodIssueOptions,
```

Comment:
dev: nothing to see here - I just made it more hipster style and export everything in one take.